[2024-01-31 15:49:11,618] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_10.dim_case scheduled__2024-01-30T15:48:42.677506+00:00 [queued]>
[2024-01-31 15:49:11,641] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_10.dim_case scheduled__2024-01-30T15:48:42.677506+00:00 [queued]>
[2024-01-31 15:49:11,642] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 15:49:11,643] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-01-31 15:49:11,646] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 15:49:11,727] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): dim_case> on 2024-01-30 15:48:42.677506+00:00
[2024-01-31 15:49:11,760] {standard_task_runner.py:52} INFO - Started process 8302 to run task
[2024-01-31 15:49:11,767] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'project_6_10', 'dim_case', 'scheduled__2024-01-30T15:48:42.677506+00:00', '--job-id', '122', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpdy0gegyv', '--error-file', '/tmp/tmpbk74zi8m']
[2024-01-31 15:49:11,770] {standard_task_runner.py:80} INFO - Job 122: Subtask dim_case
[2024-01-31 15:49:12,389] {task_command.py:369} INFO - Running <TaskInstance: project_6_10.dim_case scheduled__2024-01-30T15:48:42.677506+00:00 [running]> on host 8ebb6ccf62d7
[2024-01-31 15:49:12,726] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project_6_10
AIRFLOW_CTX_TASK_ID=dim_case
AIRFLOW_CTX_EXECUTION_DATE=2024-01-30T15:48:42.677506+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-30T15:48:42.677506+00:00
[2024-01-31 15:49:12,890] {base.py:68} INFO - Using connection ID 'postgres_dwh' for task execution.
[2024-01-31 15:49:12,987] {dbapi.py:208} INFO - Running statement: CREATE TABLE IF NOT EXISTS case_table (
    id INTEGER PRIMARY KEY,
    status_name VARCHAR(255) NOT NULL,
    status_detail VARCHAR(255) NOT NULL
), parameters: None
[2024-01-31 15:49:13,248] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=project_6_10, task_id=dim_case, execution_date=20240130T154842, start_date=20240131T154911, end_date=20240131T154913
[2024-01-31 15:49:13,489] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-31 15:49:13,921] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
